services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str | None = None) -> MovieSession:
+    if session_date:
+        date_object = datetime.strptime(session_date, "%Y-%m-%d").date()
```

Comment:
you can pass date without conversion here - django will take care of everything under the hood